src/coreclr/src/jit/lowerxarch.cpp

```diff
@@ -5464,9 +5464,29 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
                             break;
                         }
 
+                        case NI_SSE2_ShiftLeftLogical128BitLane:
+                        case NI_SSE2_ShiftRightLogical128BitLane:
+                        case NI_AVX2_ShiftLeftLogical128BitLane:
+                        case NI_AVX2_ShiftRightLogical128BitLane:
+                        {
+#if DEBUG
+                            // These intrinsics should have been marked contained by the general-purpose handling earlier in the method.
```

Comment:
This is causing CI to fail in the formatting job.  It failed on this PR as well.

Reply:
Looks like you fixed this in https://github.com/dotnet/runtime/pull/44197, 👍